src/modm/communication/xpcc/dispatcher.cpp

```diff
@@ -46,17 +60,12 @@ xpcc::Dispatcher::update()
 		{
 			const bool is_response = this->handlePacket(header, payload);
 			if (!header.isAcknowledge && header.destination != 0)
-            {
-            	if (is_response or postman->isComponentAvailable(header.destination))
-                    this->sendAcknowledge(header);
-            }
+			{
+				if (is_response or postman->isComponentAvailable(header.destination))
+					this->sendAcknowledge(header);
+			}
```

Comment:
Please avoid pure whitespace changes

Reply:
Then the indentation is consistently tabs...